src/router/PriorityFeeTunnelRouter.sol

```diff
@@ -50,6 +50,6 @@ contract PriorityFeeTunnelRouter is BaseTunnelRouter {
     }
 
     function _routerFee(uint256 gasUsed) internal view virtual override returns (uint256) {
-        return (gasFee.priorityFee + block.basefee) * gasUsed;
+        return tx.gasprice * gasUsed;
```

Comment:
ditto. also please make sure that tx.gasprice here is base+priority and it will have value if tx is eip1559.